docker-compose.yml

```diff
@@ -1,17 +1,93 @@
-version: "3"
+# this file is copied to the staging or production server to be used there
+
+version: '3'
 services:
+  app:
```

Comment:
sollen wir das auch in blitz umbennen?

Reply:
Finde `app` gar nicht so schlecht, da auch das repo `atlas-app` heißt. Es war nur verwirrend als beide container app hießen.